src/utils/getOssUserType.ts

```diff
@@ -54,6 +53,8 @@ export async function getOssUserType(
   let status: number | null;
   let check: 'Org' | 'Team';
 
+  const octokit = await getClient(ClientType.User);
```

Comment:
The second arg is dropped on the floor in the `ClientType.User` case.

https://github.com/getsentry/eng-pipes/blob/3de4f3a83da3a2a96eed03ce526492e94ba78ed0/src/api/github/getClient.ts#L36-L41